internal/flink/command_artifact_create.go

```diff
@@ -15,6 +16,18 @@ import (
 	"github.com/confluentinc/cli/v3/pkg/utils"
 )
 
+var (
+	allowedRuntimeLanguages = map[string]any{
+		"python": struct{}{},
+		"java":   struct{}{},
+	}
+	allowedFileExtensions = map[string]any{
+		"zip": struct{}{},
+		"py":  "python",
```

Comment:
we don't allow py as plugin file extension. only jar and zip for now

Reply:
Done removed.